python/dev.py

```diff
@@ -144,11 +144,16 @@ def build_async_client(release: bool, no_cache: bool = False) -> None:
     print("[OK] Async client build completed")
 
 
-def build_sync_client() -> None:
-    print("[INFO] Building sync client...")
+def build_sync_client(glide_version) -> None:
+    print(f"[INFO] Building sync client with version={glide_version}...")
     generate_protobuf_files()
 
-    run_command(["cargo", "build"], cwd=FFI_DIR, label="cargo build ffi")
+    run_command(
+        ["cargo", "build"],
+        cwd=FFI_DIR,
+        label="cargo build ffi",
+        env={"GLIDE_NAME": "GlidePyFFI", "GLIDE_VERSION": glide_version, **os.environ},
```

Comment:
move to const variables and use also for the async client, 
GLIDE_SYNC_NAME = "GlidePyFFI"
GLIDE_ASYNC_NAME = "GlidePy"

Reply:
lets use GlidePySync and not GlidePyFFI, bc the communication protocol might change in the future from FFI to another

Reply:
you can sets GLIDE_NAME env also for async client